resources/views/layouts/app.blade.php

```diff
@@ -107,25 +112,23 @@
                                         <input name="searchQuery" type="text"
                                                class="border border-white rounded-left form-control my-0 py-1"
                                                placeholder="Search" aria-label="User suchen">
-                                        <div class="input-group-append">
-                                            <button class="input-group-text btn-primary lighten-2" type="submit">
-                                                <i class="fas fa-search text-grey" aria-hidden="true"></i>
-                                            </button>
-                                        </div>
+                                        <button class="input-group-text btn-primary" type="submit">
+                                            <i class="fas fa-search" aria-hidden="true"></i>
+                                        </button>
                                     </div>
                                 </form>
                                 <li class="nav-item d-none d-md-inline-block">
                                     <a href="javascript:void(0)" id="notifications-toggle"
                                        class="nav-link notifications-board-toggle"
-                                       data-toggle="modal"
-                                       data-target="#notifications-board">
+                                       data-mdb-toggle="modal"
+                                       data-mdb-target="#notifications-board">
                                         <span class="notifications-bell far fa-bell"></span>
                                     </a>
                                 </li>
                                 <li class="nav-item dropdown">
-                                    <a id="navbarDropdown" class="nav-link dropdown-toggle mdb-select"
-                                       href="javascript:void(0)"
-                                       role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    <a id="navbarDropdown" href="#" class="nav-link dropdown-toggle mdb-select"
```

Comment:
Why do you changed this back to `href="#"?`

`javascript:void(0)` would be result in a better user experience.

Reply:
This is a remnant of #243. Won't work otherwise, afaik/iirc.